packages/web-app-files/tests/unit/components/AppBar/ViewOptions.spec.js

```diff
@@ -95,4 +97,46 @@ describe('ViewOptions', () => {
 
     expect(mockedStore.modules.Files.mutations.SET_HIDDEN_FILES_VISIBILITY).toHaveBeenCalled()
   })
+
+  it('initially shows normal resource-table by default', () => {
+    const wrapper = shallowMount(ViewOptions, {
```

Comment:
for future tests (not blocking this PR): we introduced a new way of handling mocks in unit tests. See for example https://github.com/owncloud/web/blob/5667b5fd9b58f7533d5866b27524afe3d344a59e/packages/web-app-files/tests/unit/views/spaces/DriveResolver.spec.ts#L56

The functions creating the wrappers now return an object, containing mocks, storeOptions and the wrapper. That gives you the chance to manipulate mocks within the test code.